Updating content regarding Hosting Environment

misterabdullahAziz opened the pull request:
Updating content regarding Hosting Environment. 

Closes:
- [FREYA-1208](https://scilifelab.atlassian.net/browse/FREYA-1208) 
- [FREYA-1209](https://scilifelab.atlassian.net/browse/FREYA-1209)
- [FREYA-1210](https://scilifelab.atlassian.net/browse/FREYA-1210)
- [FREYA-1236](https://scilifelab.atlassian.net/browse/FREYA-1236)

aishling-scilifelab reviewed (approved):
Did not review the Swedish text. 

Also one question on what's the advantage of using the 'draft' flag over just deleting the file and having it in the git history if we need it?

misterabdullahAziz commented on content/services/hosting.md:
```diff
@@ -2,6 +2,7 @@
 title: Hosting a service on the SciLifeLab Data Platform
 images: [/img/service_thumbnails/hosting_DC.jpg]
 toc: true
+draft: true
```
I'm sorry for not responding earlier; I missed the notification among other GitHub notifications.

Setting **`draft: true`** in Hugo prevents the page from being built and published in the public site while still keeping the content in the repository. This is useful because:
- It allows us to retain the content for future use without fully deleting it.
- We can easily enable it again by changing `draft: false` instead of recreating the file.
- It helps maintain version history and avoids accidental loss of work.

[Hugo Docs](https://gohugo.io/methods/page/draft/)

aishling-scilifelab replied:
Not a strong objection if this is your convention but git itself also does point 1 and 3 (i.e. the content is not deleted and the history is visible).

misterabdullahAziz replied:
Yes, Git provides 1 and 2.
However, with Hugo-based projects, this is common practice in team-freya.
It is also suggested by @senthil10 and I also think of this as a better way.
We can discuss this and avoid this approach by deleting pages and avoid enabling `draft`.

misterabdullahAziz commented:
I found a couple more things to fix before merging. I will reopen it when fixed.

misterabdullahAziz commented:
A couple of changes:
**About Page:**
- The previous commit resulted in similar text in the sections **About the SciLifeLab Data Platform** and **Hosting environment** so I have updated both sections. (@aishling-scilifelab please review the text if it seems fine)
- When the English text is approved, I will get the Swedish text of this and will add it.
- I removed the banner from the section **Hosting environment** and instead added a link to **Hosting Service Page**.

**Hosting Service Page:**
- I did not remove this page as it is linked to various other places such as in [news item](https://data.scilifelab.se/news/2023-04-28/) and [E-Infrastructure Page](https://data.scilifelab.se/resources/e-infrastructure/#scilifelab-data-platform).
- I kept this page with only a banner and removed all other content. Because it feels a bit dodgy to alter a news item.
- As this hosting service is not yet provided I removed the card from [Services Page](http://localhost:1313/services/) which means it cannot be accessed via services instead it can access from links provided in other places that only show a banner.
